app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,18 +11,111 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-filters-sidebar {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .background {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 200;
+      height: 100%;
+      width: 100%;
+      background-color: rgba(0, 0, 0, 0.15);
+      opacity: 0;
+      transition: opacity 250ms ease-in-out 0s;
+    }
+
+    &.shown {
+      .background {
+        opacity: 1;
+      }
+
+      .sidebar, .sidebar-footer {
+        margin-right: 0;
+      }
+    }
+
+    .sidebar {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 210;
+      height: 100%;
+      width: $sidebar-large-width;
+      margin-right: -$sidebar-large-width;
+      background-color: rgba($white, 0.95);
+      padding: 1em;
+      transition: margin 250ms ease-in-out 0s;
+      overflow-y: scroll;
+
+      .property-selectors {
+        margin-bottom: $sidebar-footer-height + 2em;
+      }
+    }
+
+    .sidebar-footer {
+      background-color: $grey-800;
+      width: $sidebar-large-width;
+      margin-right: -$sidebar-large-width;
+      height: $sidebar-footer-height;
+      position: fixed;
+      bottom: 0;
+      right: 0;
+      transition: margin 250ms ease-in-out 0s;
```

Comment:
it's the third time I see the transition, so IMO the `250ms ease-in-out 0s` it needs to be abstracted into a var so we enforce consistency.